_posts/2017-12-11-A-note-on-fees.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: A note on fees
+tags: [crypto, mrl, dev]
```

Comment:
Please change tags to: `[crypto, research]`
There exist no `mrl` or `dev` tags.

Reply:
Fixed.